bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Mocks/Mocks/SystemRegister/RegisterClientMock.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Altinn.Authentication.UI.Core.SystemRegister;
+using Altinn.Platform.Register.Models;
+
+namespace Altinn.Authentication.UI.Mocks.SystemRegister;
+public class RegisterClientMock : IRegisterClient
+{
+    public Task<Party> GetPartyForOrganization(string organizationNumber)
+    {
+        throw new NotImplementedException();
```

Comment:
i assume this will be added in this PR later

Reply:
will wait, we focus on Delegation now, because of summer vacation